tsl/test/sql/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ set(TEST_FILES_DEBUG
   deparse.sql
   dist_commands.sql
   dist_compression.sql
+  dist_continuous_aggs.sql
```

Comment:
Just so you know and don't have to spend time debugging if you run into this: I have seen failures on CI when multiple dist tests create data nodes with the same name. Renaming the data nodes to something unique to the test file usually resolves this.